### ISSUE TITLE
Brave browser tab becomes a floating window when pulled out

### ISSUE THREAD

**Issue by 27medkamal** [nikitabobko/AeroSpace#340]
First of all, want to say thanks for this great window manager. Much better than the alternatives.

# Issue
When I have a Brave browser window with multiple tabs, if I use the mouse to click on a tab and pull it out of the current window, 2 things can happen:
1. If I do it quickly and let go of the mouse as soon as the tab is out of the current window, it works fine and joins the current layout.
2. If I delay it by anything around a second or more, it ends up as a floating window, then I have to join it into the current layout with `layout floating tiling`

Correction: had this issue reported for Chrome, but the issue doesn't happen in chrome. Seems isolated to Brave. Edited the above to reflect the actual issue.

**Comment by jakenvac** [comment]
I'm also experiencing this with Brave. I think the app id for brave (`com.brave.Browser`) just needs to be added here:
https://github.com/nikitabobko/AeroSpace/blob/9faadae61776a5ae352a334ee257365b81dbdaa5/Sources/AppBundle/tree/MacWindow.swift#L239-L253

I'm unable to build and test it myself at the moment but I'll see if I can get some time later on and submit a PR.

**Comment by nikitabobko** [comment]
@jakenvac you are technically correct. We can just add Brave in the list here (similar to Chrome) and AeroSpace will force tile all Brave windows by default

I'm a little bit hesitant to do so because the solution for Chrome is not perfect:
- #332 
- #246 

Unless Brave doesn't use dialogs in their UI that will mistakenly get tiled (Chrome doesn't use dialogs at all), I'm ok with adding Brave to the list

**Comment by nikitabobko** [comment]
Right now, there is a workaround:
```
[[on-window-detected]]
if.app-id = 'com.brave.Browser'
run = 'layout tiling'
```

**Comment by 27medkamal** [comment]
I agree. I think Brave has the exact same window behaviour as Chrome. I believe it should be safe to add it.

Workaround works as well, so thanks for that.

**Comment by stevo-knievo** [comment]
I have the same issue.
But the workaround works well. Thank you!

AeroSpace v0.14.2-Beta 0cb8dbdfc5ee73b8cbc200f175f467ebead55201

PS:
@nikitabobko
I just found out about AeroSpace.
I'm my opinion best tiling window manager for macOS by far!! Thank you so so much! Time to learn Swift to contribute to the awesome project as well. Thanks again!

**Comment by nikitabobko** [comment]
Closing in favor of https://github.com/nikitabobko/AeroSpace/issues/1001
